docs/gen-docs/kyma_alpha_delete.md

```diff
@@ -16,6 +16,7 @@ kyma alpha delete [flags]
 
 ```bash
       --concurrency int              Number of parallel processes (default 4)
+      --keep-crds                    Keep CRDs during deletion.
```

Comment:
We should be consistent and match the other flags' descriptions in structure and style.
      --keep-crds                    Flag specifying whether to keep CRDs during deletion
```